source/views/sis/student-work/detail.android.js

```diff
@@ -137,6 +138,16 @@ function LastUpdated({when}: {when: string}) {
 	) : null
 }
 
+function shareJob(job: JobType) {
+	const jobBaseUrl =
+		'https://www.stolaf.edu/apps/stuwork/index.cfm?fuseaction=Details&jobID='
+	Share.share({
+		message: `${jobBaseUrl}${job.id}`,
+	})
+		.then(result => console.log(result))
```

Comment:
Remove the logging please

Reply:
If that's the case, we should remove it from calendar, too.

https://github.com/StoDevX/AAO-React-Native/blob/97b5a4e38e1aa2f506ef0c1f134838a3820d903c/source/views/calendar/calendar-util.js#L17-L19

Reply:
Yes, we should, but in another PR.